README.md

```diff
@@ -265,7 +265,7 @@ Quick start
 
 > NOTE: While kpatch is designed to work with any recent Linux
 kernel on any distribution, the `kpatch-build` command has **ONLY** been tested
-and confirmed to work on Fedora 20, RHEL 7, Oracle Linux 7, CentOS 7 and Ubuntu 14.04.
+and confirmed to work on Fedora {20,21,23}, RHEL 7, Oracle Linux 7, CentOS 7 and Ubuntu 14.04.
```

Comment:
Good point, It can be misleading. I personally haven't tested `kpatch` on Fedora 22 (technically it should, just don't want to take it for granted) that's why I didn't put it in the list. Let me know if any of you have tested on Fedora 22.

If so, I'll add 1 more commit to the PR ;-D